subworkflows/local/profiling.nf

```diff
@@ -145,7 +145,15 @@ workflow PROFILING {
     if ( params.run_kraken2 && params.run_bracken ) {
         // Remove files from 'pure' kraken2 runs, so only those aligned against Bracken & kraken2 database are used.
         def ch_kraken2_output = KRAKEN2_KRAKEN2.out.report
-            .filter { meta, report -> meta['tool'] == 'bracken' }
+            .filter {
+                meta, report ->
+                    if ( meta['instrument_platform'] == 'OXFORD_NANOPORE' ) log.warn "[nf-core/taxprofiler] Bracken has not been evaluated for Nanopore data. Skipping Bracken for sample ${meta.id}."
```

Comment:
It seems to me that you should first check for tool == bracken before issuing the warning.

Reply:
Did you test the new version? I always thought that the last value from a closure was returned.